### PR TITLE
fix: use new forked package of json-pointer that fixed critical Prototype pollution

### PR DESCRIPTION
**Pull request by francis04j**
## What/Why/How?
+--------------+------------------+----------+-------------------+---------------+---------------------------------------+
|   LIBRARY    | VULNERABILITY ID | SEVERITY | INSTALLED VERSION | FIXED VERSION |                 TITLE                 |
+--------------+------------------+----------+-------------------+---------------+---------------------------------------+
| json-pointer | CVE-2021-23820   | CRITICAL | 0.6.1             |               | Prototype Pollution in json-pointer   |
|              |                  |          |                   |               | -->avd.aquasec.com/nvd/cve-2021-23820 |
+--------------+------------------+----------+-------------------+---------------+---------------------------------------+

## Reference
https://github.com/advisories/GHSA-v5vg-g7rq-363w

### CODE REVIEW

**Comment by MichaelMcAleer** [comment]
This vulnerability has been flagged by our internal security platforms and code scanners, it would be great if this PR could go in ASAP! Thanks @francis04j for opening this request!

**Comment by RomanHotsiy** [comment]
@francis04j I can't find any changes in your forked repo. This is somehow concerned as it means you published to npm from local.

Could you push you changes?

**Comment by francis04j** [comment]
> @francis04j I can't find any changes in your forked repo. This is somehow concerned as it means you published to npm from local.
> 
> Could you push you changes?

@RomanHotsiy changs pushed

**Review by RomanHotsiy — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -88,6 +88,6 @@
   },
   "dependencies": {
     "@types/json-schema": "^7.0.7",
-    "json-pointer": "^0.6.1"
+    "json-pointer-community": "^0.7.0"
```
Comment (RomanHotsiy) [inline package.json:91]:
```suggestion
    "json-pointer-community": "0.7.0"
```

**Pull request merged**